.github/workflows/release.yml

```diff
@@ -52,6 +54,8 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
         run: |
+          set -opipefail
```

Comment:
```suggestion
          set -o pipefail
```

Reply:
Oh, I didn't know that we need a space here.